client/src/pages/Profile.jsx

```diff
@@ -1,14 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
+import { getAttendingSessions } from '../utils/apiWrapper';
+import 'semantic-ui-css/semantic.min.css';
+import '../css/Profile.scss';
+import SessionSummary from '../components/SessionSummary';
+import ClassForm from '../components/ClassForm.jsx';
 import ClassCard from '../components/ClassCard';
 
 function Profile({ user }) {
+  const [sessions, setSessions] = useState([]);
+  const [classes, setClasses] = useState([]);
+
+  useEffect(() => {
+    const populateSessionsAndClasses = async () => {
+      console.log(user);
+      const resp = await getAttendingSessions(user._id);
+      if (!resp.error) {
+        console.log(resp.data.result);
+        setSessions(resp.data.result);
+      }
+      setClasses(user.classes);
+    };
+
+    populateSessionsAndClasses();
+  }, [user]);
+
   return (
     <>
-      <h1>
-        Profile Page for {user.firstName} {user.lastName}
-      </h1>
-      <ClassCard classCardText={'CS125'} />
+      <h2>Welcome to your Profile Page, {user.firstName}</h2>
+
+      <h1>My Sessions</h1>
+      <p></p>
+      {sessions.map((session, i) => (
+        <SessionSummary user={user} session={session} key={i} />
+      ))}
+
+      <h1>My Classes</h1>
+      <div className="classes-display">
+        {classes &&
+          classes.map(
+            (userClass, j) =>
+              classes && <ClassCard classCardText={userClass} key={j} />,
```

Comment:
You can use `i` again here even though you used it above because they're in different scopes